### PR TITLE
Add Screen warnings for large number of controls and use if watchers/scripts

### PR DESCRIPTION
**Pull request by caleeli**
Fixes: https://processmaker.atlassian.net/browse/FOUR-3698

Add Screen warnings for large number of controls and use if watchers/scripts

ScreenWarningns2

Example:
[LargeScreen.json.txt](https://github.com/ProcessMaker/processmaker/files/6895994/LargeScreen.json.txt)

### CODE REVIEW

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://bugfix-four-3698-processmaker.ci.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://bugfix-four-3698-processmaker.ci.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://bugfix-four-3698-processmaker.ci.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://bugfix-four-3698-processmaker.ci.processmaker.net

File: resources/js/processes/screen-builder/screen.vue [hunk 1]
```diff
@@ -428,13 +447,37 @@ export default {
...
+        .filter(watcher => watcher.script.id.substr(0, 7) === 'script-').length;
+      if (watchersWithScripts > 0) {
+        warnings.push({
+          message: this.$t('Using watchers with Scripts can slow the performance of your screen.'),
+        });
+      }
+      // Count form elements
+      if (this.numberOfElements >= 25) {
+        warnings.push({
+          message: this.$t('We recommend using fewer than 25 form elements in your screen for optimal performance.'),
```
Comment (danloa) [inline resources/js/processes/screen-builder/screen.vue:464]: Add this string to translations

File: resources/js/processes/screen-builder/screen.vue [hunk 2]
```diff
@@ -428,13 +447,37 @@ export default {
...
-    }
+    },
+    warnings() {
+      const warnings = [];
+      // Check if screen has watchers that use scripts
+      const watchersWithScripts = this.watchers
+        .filter(watcher => watcher.script.id.substr(0, 7) === 'script-').length;
+      if (watchersWithScripts > 0) {
+        warnings.push({
+          message: this.$t('Using watchers with Scripts can slow the performance of your screen.'),
```
Comment (danloa) [inline resources/js/processes/screen-builder/screen.vue:458]: Add this string to translations

**Review by danloa — CHANGES_REQUESTED** [review]

File: resources/js/processes/screen-builder/screen.vue (hunk 1 above)
Comment (caleeli) [reply resources/js/processes/screen-builder/screen.vue:464]: Missing translations were added

File: resources/js/processes/screen-builder/screen.vue (hunk 2 above)
Comment (caleeli) [reply resources/js/processes/screen-builder/screen.vue:458]: Missing translations were added

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://bugfix-four-3698-processmaker.ci.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://bugfix-four-3698-processmaker.ci.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://bugfix-four-3698-processmaker.ci.processmaker.net

**Review by danloa — APPROVED** [review]

**Review by boliviacoca — APPROVED** [review]

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://bugfix-four-3698-processmaker.ci.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://4-1-21-processmaker.ci.processmaker.net

**Pull request merged**